regulament_pregatit_feedback.txt

```diff
@@ -660,6 +665,79 @@ Art. 106.
 
 Art. 107. Centrele Locale care nu se supun procedurilor de desființare vor răspunde în justiție.
 
+Art. 107+1. Centrele Locale pot mentora unul sau mai multe Grupuri de Cercetași, în următoarele condiții:
+	(1) Numărul de membri al Grupurilor de cercetași, în total, nu depășește 1/2 din numărul total de membri ai Centrului Local la momentul asumării Planului de Mentorat.
+	(2) Centrul Local a obținut acreditarea de Centru Local Mentor de la Consiliului Director
+
+Art. 107+1. Acreditarea de Centru Local Mentor este conferită Centrelor Locale, la cerere, de către Consiliul Director, în urmă verificării următoarelor condiții:
+	(1) Centrul Local are lideri care au experiență relevantă și actuală în lucrul cu copii și tineri și în viața Centrului Local și care au parcurs, cu maxim un an înaintea obținerii acreditării, stagiul pentru mentoratul Grupurilor de Cercetași
+	(2) Dacă Centrul Local mentorează sau a mentorat în trecut Grupuri de Cercetași, acesta desfășoară / a desfășurat responsabil procesele respective, îndeplinind cu bună credință acțiunile din Planurile de Mentorat
+	(3) Centrul Local nu este sub efectul unei sancțiuni și are un istoric care îl recomandă pentru a fi mentor pentru Grupuri de Cercetași
+	(4) Centrul Local îndeplinește criteriile minime IEDLOC pentru mentorat
+
```

Comment:
Ce mai e menționat în documentul nostru (https://consiliuldirector.scout.ro/2022/03/09/acordarea-dreptului-de-mentorat/) e Demonstrează o bună guvernanță. Adăugăm și asta aici?

Reply:
Pare destul de arbitrar. Nu e ceva ce poate fi masurat si monitorizat in IEDLOC pentru mentorat?